modules/src/ics03_connection/connection.rs

```diff
@@ -18,13 +20,12 @@ pub struct ConnectionEnd {
 }
 
 impl TryFromRaw for ConnectionEnd {
-    type RawType = RawConnectionEnd;
+    type RawType = RawIdentifiedConnection;
     type Error = anomaly::Error<Kind>;
-    fn try_from(value: RawConnectionEnd) -> Result<Self, Self::Error> {
+    fn try_from(value: RawIdentifiedConnection) -> Result<Self, Self::Error> {
```

Comment:
I looked for where is this conversion used and could not find. Can you point me to where do we use the decoding of RawIdentifiedConnection -> ConnectionEnd ? Is it the call to `query` from `CosmosSDKChain`?

Reply:
yes it was from the query to fix the connection query with Gaia/ Go Relayer. But as it turns out I was using a slightly older version and once I updated the connection query worked fine. I just reverted this change.